torch/_torch_docs.py

```diff
@@ -8019,28 +8048,25 @@ def merge_dicts(*dicts):
           algorithm) instead of `?gesvd` for speed. Analogously, the SVD on GPU uses the MAGMA routine
           `gesdd` as well.
 
-.. note:: Irrespective of the original strides, the returned matrix `U`
-          will be transposed, i.e. with strides :code:`U.contiguous().transpose(-2, -1).stride()`
+.. note:: The returned matrix `U` will be transposed, i.e. with strides
+          :code:`U.contiguous().transpose(-2, -1).stride()`.
 
-.. note:: Extra care needs to be taken when backward through `U` and `V`
-          outputs. Such operation is really only stable when :attr:`input` is
-          full rank with all distinct singular values. Otherwise, ``NaN`` can
-          appear as the gradients are not properly defined. Also, notice that
-          double backward will usually do an additional backward through `U` and
-          `V` even if the original backward is only on `S`.
+.. note:: Gradients computed using `U` and `V` may be unstable if
```

Comment:
"Unstable" is a vague term. Can we make this more precise? Alternatively, maybe we should remove this note.

Reply:
Honestly, I don't know how to improve. But note that this sentence is yours, see https://github.com/pytorch/pytorch/pull/45562#discussion_r524773439 ;)

Reply:
Maybe write it as cause and effect. Something like "If :attr:`input` is not full rank or has non-unique singular values the gradients computed using `U` and `V` may be unstable, meaning ...". I'm not sure what unstable here means either so maybe we should add an explanation for that.

Reply:
> Honestly, I don't know how to improve. But note that this sentence is yours, see #45562 (comment) ;)


I am my own harshest critic. Let's just leave it for now and we can revisit it later.